docs/how-to-install.md

```diff
@@ -0,0 +1,15 @@
+### Installation
```

Comment:
We'd better list all of deps that's needed for building.
Such as bazel in 4.2.1 and higher version.
gcc-9
jdk-8, etc.

Reply:
add basic env requirement in doc